src/main/java/com/searchspring/nextopia/model/ParameterMappings.java

```diff
@@ -0,0 +1,6 @@
+package com.searchspring.nextopia.model;
+
+public class ParameterMappings {
+    public final static String NX_KEYWORDS = "keywords";
+    public final static String SS_KEYWORDS = "query";
```

Comment:
I might be tempted to have this class envelope an actual map, which can be generated by a local JSON file or some such thing via method or `init`.

Reply:
I imagine that this class is going to have some logic in it to parse specific types of parameters that aren't an exact copy the value across.  Like the weirdness with the searchspring ordering of some parameters.